packages/grafana-toolkit/docker/grafana-plugin-ci/README.md

```diff
@@ -31,21 +31,23 @@ There are 4 previous versions pre-downloaded to /usr/local/grafana. These versio
 3. 6.4.5
 4. 6.3.7
 
-To test, your circle-ci config will need a run section with something similar to the following
+To test, your CircleCI config will need a run section with something similar to the following
 ```
 - run:
         name: Setup Grafana (local install)
         command: |
           sudo dpkg -i /usr/local/grafana/deb/grafana_6.6.2_amd64.deb
+          sudo updatedb
+          sudo locate grafana
```

Comment:
Why are these commands necessary? It will just print the location of grafana?

Reply:
Well, they may not be, but at the time it was taking a long time to commit a new VM. Upon review, I had not updated the readme to match what I had tested. Yeah, this may not be necessary, but taking them both out caused an error, and in the interest of time decided not to pick that battle. I can still figure it out, I'm just competing for bandwidth with kids as they are all home youtubing :)